.github/workflows/build-nomad-spark-distro.yaml

```diff
@@ -0,0 +1,42 @@
+name: Build and Test Nomad-Spark Distribution
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to Cut
+        default: 2.4.3
+        required: true
+      nomadVersion:
+        description: Nomad Version used for Cutting
+        default: 0.8.6
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Tag/Build/Test
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: '1.8'
+      - name: Cut Tag from Branch
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+          NOMAD_VERSION: ${{ github.event.inputs.nomadVersion }}
+        run: git tag $VERSION-nomad-$NOMAD_VERSION-$(date +'%m%d%Y')
```

Comment:
we are using 0.8.6 nomad. 
why cant we just hard code here ?

Reply:
lets use the same value for `VERSION` that `make-distribution.sh` comes up with.

Reply:
Can you give us an example value for the tag that gets built

Reply:
Are we _always_ using 0.8.6? Is there utility for changing it? I had it as a variable as I figured it was subject to change like the branch version. As for a tag value, it'd be something like "2.4.3-nomad-0.8.6-08112020"

Reply:
make-distribution comes later in the build, unless you mean to re-order it or to pull some logic from that script out.